src/main/java/VladMaltsev/weatherapp/entity/WeatherDaySnapshot.java

```diff
@@ -0,0 +1,43 @@
+package VladMaltsev.weatherapp.entity;
+
+import jakarta.persistence.*;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.format.annotation.DateTimeFormat;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@Entity
+@Table(name = "weather_summary")
+@Data
+@NoArgsConstructor
+public class WeatherDaySnapshot {
+
+    @Id
+    @Column(name = "id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    @DateTimeFormat(iso = DateTimeFormat.ISO.DATE)
+    @Column(name = "date")
+    private LocalDate date;
+
+    @Column(name = "address")
+    private String address;
```

Comment:
It's ok for prototype but there's probably a better type for address here. Also, what's the logic for address, how is it used? Is this how weather service API works?  usually weather is bound to GPS coordinates I think.